### PR TITLE
Wrote preliminary version of README.md

### PR DESCRIPTION
**Pull request by NickMuise**
Now others can comment on pull request to help create README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
-# skate-project
-Machine-Perception skeet skeet
+﻿# TODO Name of project other than Skateboard Ranch
```
Comment (tlacasse) [inline README.md]: how about "Haar Cascade Training Data Preprocessing/Simplification" or something like that

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+A machine perception project using Python/OpenCV to attempt to streamline the processes of gathering training data for a Haar Cascades classifier on skateboards and actually use said classifiers to attempt to classify given images. 
+Although this project was initially created to filter/detect skateboards from video files, but was designed with reusability and reapplication of code to new purposes in mind.
```
Comment (tlacasse) [inline README.md]: missshapen sentence
```suggestion
Although this project was initially created to filter/detect skateboards from video files, it was designed with reusability and reapplication of code to new purposes in mind.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+
+## Motivation
+
+This project was originally created as the final project for Machine Perception Spring 2019.
+
+## Screenshots
+
+TODO screenshot of it drawing a rectangle over skateboard
```
Comment (tlacasse) [inline README.md]: i will add a gif

File: README.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+
+## Features
+
+This project attempts to streamline the often-laborious process of defining regions of interest in sets of unclassified training data, i.e. the process of cropping thousands of unisolated images of the object of interest when a freely-available, labeled training dataset for that object is not available. E.g., this project attempts to remove the tedium of cropping thousands of images of skateboards when trying to acquire a training dataset for a Haar Cascade skateboard classifier.
```
Comment (tlacasse) [inline README.md]: maybe do this as a list

Comment (tlacasse) [reply README.md]: maybe also add
```md
  *rest of text*. Includes:
     - filtering a mask out of original image using color (with filter_finder utility)
     - automatically determining cropping bounds using mask.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,49 @@
...
+TODO other parts of the project
+
+## Code Example
```
Comment (tlacasse) [inline README.md]: we'll have examples in how to use.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,49 @@
...
+TODO
+
+## Installation/Setup
+
+This project requires the following to be installed:
+
+-Python 2.7.15 or higher (not tested in Python 3)
+-cv2 3.4.5 or higher
+
+## Tests
```
Comment (tlacasse) [inline README.md]: we dont have tests

File: README.md [hunk 7]
```diff
@@ -1,2 +1,62 @@
...
+> python FrameRipper.py VIDEOSPATH [--framecount FRAMECOUNT] 
+
+Arguments:
+- `videospath` - path to videos to rip and crop frames
+- `framecount` *(optional)* - number of frames to rip for each video
+
+
+## Credits
+
+We would like to thank Ali Parlakci's Bulk Downloader for Reddit, https://aliparlakci.github.io/bulk-downloader-for-reddit/, for enabling us to easily get a large set of negative images by bulk-downloading images from /r/pics.
```
Comment (tlacasse) [inline README.md]:
```suggestion
We would like to thank [Ali Parlakci's Bulk Downloader for Reddit](https://aliparlakci.github.io/bulk-downloader-for-reddit/) for enabling us to easily get a large set of negative images by bulk-downloading images from /r/pics.
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,62 @@
...
+We would also like to thank the Cascade Trainer GUI, http://amin-ahmadi.com/cascade-trainer-gui/, for allowing us to easily and and quickly train several Haar Cascade classifiers.
```
Comment (tlacasse) [inline README.md]:
```suggestion
We would also like to thank the [Cascade Trainer GUI](http://amin-ahmadi.com/cascade-trainer-gui/) for allowing us to easily and quickly train several Haar Cascade classifiers.
```

**Review by tlacasse — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (NickMuise) [reply README.md]: yeah we don't

File: README.md (hunk 4 above)
Comment (NickMuise) [reply README.md]: yes

**Pull request merged**